### ISSUE TITLE
waveSharp bug/error : 

### ISSUE THREAD

**Issue by PaulJonesnonotthatone** [CorBer/waveSharp#25]
**Describe the bug**
Wavesharp no longer opens on my Windows machine. I used it a few times in July, traveled, and now it won't open. It hangs up and shows an icon on the taskbar, but never opens.

**To Reproduce**
Steps to reproduce the behavior from the start of waveSharp.
1. try to open waveSharp (double click, right click, or run as administrator)
2. Nothing happens 

**Expected behavior**
Program opens

**Screenshots**
 N/a I don't think.

**Operating system and version**
Windows 10 Home, version 22H2

**Additional context**
I suspect a recent windows update. I've deleted and reinstalled and get the same error. Thanks.

**Comment by CorBer** [comment]
Hi Paul,

I cannot reproduce that error as I am not using a recent windows on any of my machines. Have you tried re-installing ? cheers Cor

**Comment by PaulJonesnonotthatone** [comment]
Hi Cor,
This is embarrassing. I didn't realize what the problem was. It may be a bug, but I'm not sure. When I had last used Wavesharp, I had an external monitor. When I tried to open and thought I failed, the laptop was projecting it to the external monitor, which I wasn't connected to at the time. This morning, when I hooked back up to the monitor to work on the problem, it opened fine. I don't know if this is wavesharp behaving oddly or windows. In any case, the program opens. I just have to be sure to move it back to the laptop screen before closing it.
